src/content/platform-integration/android/restore-state-android.md

```diff
@@ -158,11 +143,10 @@ check out the following resources:
 * For more information on navigation and the
   `go_router` package, check out [Navigation and routing][].
 
+[`go_router`]: {{site.pub}}/packages/go_router
```

Comment:
since we mentioned go_router in this doc, we should also mention this guide https://pub.dev/documentation/go_router/latest/topics/State%20restoration-topic.html